### PR TITLE
WFCORE-5810 Fix license in wildfly-cli jar.

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFCORE-5810

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
@jfdenise Can you please review the PR

File: core-feature-pack/common/src/main/resources/license/core-feature-pack-common-licenses.xml [hunk 1]
```diff
@@ -230,6 +230,11 @@
           <url>http://www.apache.org/licenses/LICENSE-2.0</url>
           <distribution>repo</distribution>
         </license>
+        <license>
```
Comment (jfdenise) [inline core-feature-pack/common/src/main/resources/license/core-feature-pack-common-licenses.xml:233]: @RanabirChakraborty , the same license needs to be added to wildfly-cli artifact in the same file:
https://github.com/wildfly/wildfly-core/blob/dd456d87dcbf03d6bbc3db05d48b1d7f38e92e0f/core-feature-pack/common/src/main/resources/license/core-feature-pack-common-licenses.xml#L582

**Review by jfdenise — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -51,6 +51,11 @@
             <url>http://repository.jboss.org/licenses/apache-2.0.txt</url>
             <distribution>repo</distribution>
         </license>
+        <license>
+            <name>Eclipse Public License - v 1.0</name>
+            <url>https://www.eclipse.org/legal/epl-v10.html</url>
+            <distribution>repo</distribution>
+        </license>
     </licenses>
```
Comment (yersan) [inline pom.xml:59]: I think the licenses on the main pom.xml file are the wildfly-core license and not an aggregation of the licenses used by the modules shipped with it. "Eclipse Public License" should be removed from the main pom.xml

**Review by yersan — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (yersan) [reply pom.xml:59]: Sorry, I should have read the Jira description first. It seems pretty clear we need it here, so, ignore my comment

Comment (bstansberry) [reply pom.xml:59]: It's not clear to me, so I've put the 'hold' label on this while I get expert advice. :)

Comment (bstansberry) [reply pom.xml:59]: @RanabirChakraborty The feedback I've gotten is this is not required, so please remove it. Thanks for your patience.

**Review by bstansberry — CHANGES_REQUESTED** [review]
I've removed the 'hold' label from this but would want to see the change to the root pom removed.

File: pom.xml (hunk 2 above)
Comment (RanabirChakraborty) [reply pom.xml]: Thanks! @bstansberry did the changes :)

**Review by yersan — APPROVED** [review]
As per discussions, this looks good to me

**Review by jfdenise — APPROVED** [review]

**Review by bstansberry — APPROVED** [review]

**Pull request merged**